src/main/scala/rocket/RocketCore.scala

```diff
@@ -53,7 +53,8 @@ case class RocketCoreParams(
   mvendorid: Int = 0, // 0 means non-commercial implementation
   mimpid: Int = 0x20181004, // release date in BCD
   mulDiv: Option[MulDivParams] = Some(MulDivParams()),
-  fpu: Option[FPUParams] = Some(FPUParams())
+  fpu: Option[FPUParams] = Some(FPUParams()),
+  debugROB: Boolean = false // if enabled, uses a C++ debug ROB to generate trace-with-wdata
```

Comment:
This is fine before Chisel5, I think eventually we need split the test(unsynthesisable) codes out from RTL with the Probe API.

Reply:
We can just replace DebugROB backend with XMR instead of DPI when that time comes